src/common/tests/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+#========================================================================
```

Comment:
entirely empty?

Reply:
Shall I remove it? It'd be useful once we want to add it!

Reply:
yeah, but there's nothing in it. I'd remove it

Reply:
obviously, means removing the change in `common/CMakeLists.txt` as well